x/blob/ante/blob_share_decorator_test.go

```diff
@@ -124,11 +136,12 @@ func TestMaxTotalBlobSizeAnteHandler(t *testing.T) {
 			require.NoError(t, txBuilder.SetMsgs(tc.pfb))
 			tx := txBuilder.GetTx()
 
-			mbsd := ante.NewMaxBlobSizeDecorator(mockBlobKeeper{})
+			mbsd := ante.NewBlobShareDecorator(mockBlobKeeper{})
 			_, err := mbsd.AnteHandle(ctx, tx, false, mockNext)
 
 			if tc.wantErr {
 				assert.Error(t, err)
+				assert.ErrorIs(t, err, blob.ErrBlobTooLarge)
```

Comment:
[suggestion-optional] For better readability, the error type could have been part of the `testCase` struct.

Reply:
Good point, resolved in 5bdc057d4a24c248c72e24767ed8dfca202eed60